ResearchKit/Localized/en.lproj/ResearchKit.strings

```diff
@@ -443,6 +443,16 @@
 "TREMOR_SKIP_RIGHT_HAND" = "I cannot perform this activity with my RIGHT hand.";
 "TREMOR_SKIP_NEITHER" = "I can perform this activity with both hands.";
 
+/* Trail making task. */
+"TRAILMAKING_TASK_TITLE" = "Trail Making Test";
+"TRAILMAKING_INTENDED_USE" = "This activity evaluates your visual attention and task switching by recording the time required to tap a series of dots in order.";
+"TRAILMAKING_INTENDED_USE2" = "After the countdown, tap dots in alternating order between numbers and letters. Begin by tapping the first number '1' followed by tapping the first letter 'A', and then '2' – 'B' – '3' – 'C'… until you reach the number '7'.";
```

Comment:
Should there be separate instructions for the other trail type (numbers-only)? Or does this line just get left out in that case?